k0s docs update (#733)
k0sproject/k0s: docs/k0s-reset.md

# k0s reset

k0s Command-Line interface now allow users to entirely remove all k0s related files from the host.

> The reset command operates under assumption that k0s is running as service on the host. So first we need to stop the service:

Reviewer: ```suggestion
The reset command operates under assumption that k0s is running as service on the host. and to prevent accidental triggering, it will not run if the k0s service is up. So first, we need to stop the service:
```

Reviewer 2: `as a service`

> The reset command operates under assumption that k0s is running as service on the host. So first we need to stop the service:

Reviewer 2: ```suggestion
The reset command operates under assumption that k0s is running as a service on the host. So first we need to stop the service:
```

> ```
> $ systemctl stop k0scontroller.service // in the case of worker node just change service name to k0sworker.service
> ```

Reviewer 2: ```suggestion
For controller nodes:

```sh
$ systemctl stop k0scontroller
```
For worker nodes:
```sh
$ systemctl stop k0sworker
```

```

> Is service is not stopped k0s will return error message:

Reviewer 2: ```suggestion
Is the service is not stopped k0s will return an error message:
```

> ```

Reviewer 2: ```suggestion
```sh
```

$ k0s reset
FATA k0s seems to be running! please stop k0s before reset. 
```

> ```

Reviewer 2: ```suggestion
If k0s has been stopped, the reset operation will proceed:
 
```sh
```

# k0s reset

> k0s Command-Line interface now allow users to entirely remove all k0s related files from the host.

Reviewer 3: -> "k0s command line interface allows users to remove all..."

The reset command operates under assumption that k0s is running as a service on the host. and to prevent accidental triggering, it will not run if the k0s service is up. So first, we need to stop the service:

For controller nodes:

```sh
$ systemctl stop k0scontroller

> Is the service is not stopped k0s will return an error message:

Reviewer 2: ```suggestion
If the service is not stopped k0s will return an error message:
```

# k0s reset

k0s Command-Line interface allows users to remove all k0s related files from the host.

> The reset command operates under assumption that k0s is running as a service on the host. and to prevent accidental triggering, it will not run if the k0s service is up. So first, we need to stop the service:

Reviewer 2: ```suggestion
The reset command operates under the assumption that k0s is installed as a service on the host. To prevent accidental triggering, it will not run if the k0s service is running. So first, we need to stop the service:
```
